lib/travis/api/v3/models/user_settings.rb

```diff
@@ -0,0 +1,8 @@
+module Travis::API::V3
+  class Models::UserSettings < Travis::Settings::Model
+    attribute :builds_only_with_travis_yml, Boolean, default: false
```

Comment:
Would be cool to set these user settings up as a first class record (via Virtus) in the same way as env vars.